CHANGELOG.md

```diff
@@ -4,6 +4,38 @@ All notable changes to this project will be documented in this file.
 
 The format follows [keepachangelog.com]. Please stick to it.
 
+## [2.6.2 Toothless Taipan] -- unreleased
```

Comment:
Nice name. :smile:
We had the habit that we only give minor releases a new name (that's only a patch release).
I'd be fine to make a minor release though (i.e. increment to 2.7.0).

Reply:
Yes there are enough external changes to justify a minor release